pytest_fixtures/api_fixtures.py

```diff
@@ -132,13 +133,9 @@ def default_architecture():
     return arch
 
 
-@pytest.fixture(scope='session')
+@pytest.fixture(scope='module')
```

Comment:
Why the scope is changed. The `default_os` will always be the same for all the modules. So the session scope is correct.

Reply:
ahh .. typo .. it should be good in latest push now.